pyanaconda/modules/timezone/installation.py

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright (C) 2019 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+import os.path
+from pyanaconda.timezone import is_valid_timezone, write_timezone_config
+from pyanaconda import ntp
+from pyanaconda.modules.common.task import Task
+from pyanaconda.modules.common.constants.services import TIMEZONE
+from pyanaconda.anaconda_loggers import get_module_logger
+
+__all__ = ["ConfigureTimezoneTask"]
+
+log = get_module_logger(__name__)
+
+
+class ConfigureTimezoneTask(Task):
+    """Installation task for timezone configuration."""
+    def __init__(self, sysroot, timezone):
+        """Create a new task,
+        :param sysroot: a path to the root of the installed system
+        :param str timezone: time zone to set
+        """
+        super().__init__()
+        self._sysroot = sysroot
+        self.timezone = timezone
+
+    @property
+    def name(self):
+        return "Configure time zone"
+
+    def run(self):
+        # get the DBus proxies
+        timezone_proxy = TIMEZONE.get_proxy()
+
+        # write out timezone configuration
+        kickstart_timezone = timezone_proxy.Timezone
+
+        # write out timezone configuration
+        # kickstart_timezone = self.timezone
```

Comment:
Please remove commented code. In general it's very rare to have a meaningful use for a commented code.